src/browser/cordova-http-plugin.js

```diff
@@ -237,7 +237,16 @@ function sendRequest(method, withData, opts, success, failure) {
   // we can't set connect timeout and read timeout separately on browser platform
   xhr.timeout = readTimeout * 1000;
 
-  setHeaders(xhr, headers);
+  try {
+    setHeaders(xhr, headers);
+  } catch(error) {
+    return onFail({
+      status: -8,
```

Comment:
@alexander-gruenewald  Would you please use code -1 for this instead? Code -8 is meant to be used for requests which were actively aborted via the API.

Reply:
@silkimen i've changed the code from -8 to -1